package/MDAnalysis/lib/util.py

```diff
@@ -559,28 +559,26 @@ def __init__(self, stream, filename, reset=True, close=False):
         the keyword *reset* is set to ``False``. If rewinding fails, a
         :class:`MDAnalysis.StreamWarning` is issued.
 
-        .. Note::
-
-           By default, this stream will *not* be closed by :keyword:`with` and
-           :meth:`close` (see there) unless the *close* keyword is set to
-           ``True``.
-
-        Arguments
-        ---------
+        Parameters
+        ----------
         stream : stream
-               an open stream (e.g. :class:`file` or :func:`cStringIO.StringIO`)
+            an open stream (e.g. :class:`file` or :func:`cStringIO.StringIO`)
         filename : str
-               the filename that should be associated with the stream
-
-        Keywords
-        --------
-        reset : boolean, default ``True``
-               start the stream from the beginning (either :meth:`reset` or :meth:`seek`)
-               when the class instance is constructed
-        close : booelan, default ``True``
-               close the stream when a :keyword:`with` block exits or when
-               :meth:`close` is called; note that the default is **not to close
-               the stream**
+            the filename that should be associated with the stream
+        reset : boolean
+            start the stream from the beginning (either :meth:`reset` or :meth:`seek`)
+            when the class instance is constructed
+        close : booelan
```

Comment:
boolean (typo)

Also note that `bool` gets linked to [bool](https://docs.python.org/2/library/functions.html#bool) whereas `boolean` is left as text.